src/ui/settings/ViewSettingsMenu.qml

```diff
@@ -176,5 +171,38 @@ VPNFlickable {
             }
         }
     }
-}
 
+    Connections {
+        target: VPNProfileFlow
+
+        // TODO: Remove example data
+        function onPopulateFakeData() {
```

Comment:
to cleanup the code a bit more, move this to C++ and remove the signal, and make `fromJSON` not invokable.

Reply:
Moved to a dedicated method `populateFakeData` in the `SubscriptionData` class.